storybook/webpack.config.js

```diff
@@ -29,10 +29,21 @@ const scssLoaders = ( { isLazy } ) => [
 module.exports = ( { config } ) => {
 	config.module.rules.push(
 		{
-			test: /\/stories\/.+\.js$/,
+			// Currently does not work with our tsx stories
+			// See https://github.com/storybookjs/storybook/issues/17275
+			test: /\/stories\/.+\.(j|t)sx?$/,
```

Comment:
I've tried adding a Story description to `Divider` (a `tsx` story) and it seemed to work — what should we expect to be broken?

Reply:
This rule is for `@storybook/source-loader`, aka this tab:

![The "Story" tab on Storybook for the Divider component is stuck on the "1 loading source..." state](https://user-images.githubusercontent.com/555336/157023991-84869e2c-dfb1-443f-8fa4-daccaa038065.png)

It stays stuck in this loading state. Fortunately this Story tab isn't particularly necessary once we get the code snippets cleaned up in Docs view. We might even remove it at some point?

Reply:
Yeah, if we show code snippets in the docs it's not really necessary